lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1920,18 +1924,38 @@ mkApiTransactionFromInfo
     => TimeInterpreter (ExceptT PastHorizonException IO)
     -> TransactionInfo
     -> m (ApiTransaction n)
-mkApiTransactionFromInfo ti (TransactionInfo txid fee ins outs ws meta depth txtime txmeta) = do
-    apiTx <- liftIO $ mkApiTransaction ti txid fee (drop2nd <$> ins) outs ws (meta, txtime) txmeta $
-        case meta ^. #status of
+mkApiTransactionFromInfo ti info = do
+    apiTx <- liftIO $ mkApiTransaction ti
+        (txInfoId)
+        (txInfoFee)
+        (txInfoInputs <&> drop2nd)
+        (txInfoCollateralInputs <&> drop2nd)
+        (txInfoOutputs)
+        (txInfoWithdrawals)
+        (txInfoMeta, txInfoTime)
+        (txInfoMetadata) $
+        case txInfoMeta ^. #status of
             Pending  -> #pendingSince
             InLedger -> #insertedAt
             Expired  -> #pendingSince
-    return $ case meta ^. #status of
+    return $ case txInfoMeta ^. #status of
         Pending  -> apiTx
-        InLedger -> apiTx { depth = Just depth  }
+        InLedger -> apiTx {depth = Just txInfoDepth}
         Expired  -> apiTx
   where
-      drop2nd (a,_,c) = (a,c)
+    TransactionInfo
```

Comment:
Perhaps we could just use `#view` to refer to each field instead of record wild-card syntax.

Reply:
Fixed in b91d00ecde1bbc4bcbeedecf3c0534cf0b0ebf0d.